config/locales/en/transition_landing_page.yml

```diff
@@ -14,6 +14,13 @@ en:
     comms_header: "News"
     comms:
       links:
+      - link:
+          text: "Our approach to the Future Relationship with the EU"
+          path: "/government/publications/our-approach-to-the-future-relationship-with-the-eu"
+          description: "The UK’s approach to negotiations with the European Union."
+          metadata:
```

Comment:
This needs to be less indented - metadata should be on the same level as "link" (see below for examples).

Reply:
Good catch